src/ProductsList.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductsList {
+    public static void printAllProducts(List<Product> products) {
```

Comment:
no necessity to create special methods to print - it can be done in the stream chain